services-marionette/firefox-services-tests/tests/functional/pocket/test_pocket_signup_fxa.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from firefox_puppeteer.testcases.base import FirefoxTestCase
+from firefox_services_tests.apps.pocket.app import Pocket
+
+
+class TestCreateNewFxaSignIn(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.pocket_page = Pocket(self.marionette)
+        self.login_page = self.pocket_page.launch_signin()
+        # TODO: Probably have some Firefox Accounts login info
```

Comment:
That will be a second bug. This bug was more to set up the page objects. Actual test cases are now going to be filed against our test plan.